docs/release-notes/Percona-Everest-1.6.0-(2025-04-16).md

```diff
@@ -0,0 +1,160 @@
+    | **7.**|[Bugs](Percona-Everest-1.6.0-%282025-04-16%29.html#bugs)|Find out about all the bugs fixed in Percona Everest 1.6.0|
+    | **8.**|[Known limitations](https://docs.percona.com/everest/release-notes/Percona-Everest-1.6.0-%282025-04-16%29.html#known-limitation)|Discover all the known limitations in Percona Everest 1.6.0|
+
+
+## 🌟 Release highlights
+
+===  "📈 Manual storage scaling"
+
+    ### Scale your storage with ease with Percona Everest 1.6.0
+
+    Starting with Percona Everest 1.6.0, you can leverage manual storage scaling to increase the capacity of your database, offering greater control over resource allocation as your needs evolve.
+    
+    **Prerequisites for storage scaling**
+
+    - **PersistentVolumeClaim (PVC) volume expansion** - Ensure that the `StorageClass` used by the database’s PersistentVolumeClaim (PVC) supports volume expansion.  Refer to the [Kubernetes documentation on Persistent Volumes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/){:target="_blank"} for more details.
+
+    - **Resource quota check** - Verify that your resource quotas allow for the requested storage capacity. For more information, see the [Kubernetes documentation on Storage Resource Quota](https://kubernetes.io/docs/concepts/policy/resource-quotas/#storage-resource-quota){:target="_blank"} 
+
+    **How to modify storage capacity**
+
+    To increase the DISK for a database, go to the Percona Everest home page and select your desired database. Then, navigate to **Overview > Resources > Edit** and enter the new value for **DISK** (in Gi).
+
+    ![!image](../images/edit_storage_capacity.png)
+
+
+    :mag: If you want to explore this topic in depth, check out our detailed [documentation](https://docs.percona.com/everest/use/manual_storage_scaling.html)!
+
+=== "🔄 Major database upgrades"
+
+    ### Seamless major version upgrades for MongoDB
+
+    Percona Everest 1.6.0 introduces support for major version upgrades of MongoDB, enabling you to upgrade your databases with **minimal downtime and disruption**. This enhancement ensures your applications remain secure, performant, and compliant with the latest MongoDB features.
+
+    To upgrade your MongoDB database, navigate to the Percona Everest homepage and select the database you wish to upgrade. On the **Overview** page, locate the **DB Details** panel and click **Edit** next to **Basic Information**. Select the desired version and click **Upgrade** to begin the process.
+
+    ![!image](../images/database_upgrade.png)
+    
+    ![!image](../images/upgrade_database_versions.png)
+
+     :mag: For a deep dive into this topic, refer to our [documentation](https://docs.percona.com/everest/upgrade/mongodb_major_upgrades.html).
+
+
+=== ":simple-postgresql: PostgreSQL Operator"
+
+    ### Support for PostgreSQL Operator 2.6.0
+
+    Percona Everest 1.6.0 now includes support for Percona Operator for PostgreSQL 2.6.0.
```

Comment:
I think we should advertise that 2.6.0 brings along PG17. This is the thing that most users will be interest in :slightly_smiling_face: 
Something like:
```suggestion
    ### Support for PostgreSQL Operator 2.6.0 and PostgreSQL 17

    Percona Everest 1.6.0 now includes support for Percona Operator for PostgreSQL 2.6.0 and PostgreSQL 17.
```